spec/swagger_v2/namespace_tags_prefix_spec.rb

```diff
@@ -0,0 +1,168 @@
+require 'spec_helper'
+
+describe 'a simple mounted api' do
+  before :all do
+    class CustomType; end
```

Comment:
Because these are global you either want to put them into spec/support or make dynamic classes, like `let(:custom_type) { Class.new }`. Same for the API below, so `let(:api) { Class.new(Grape::API) do ... }`.